qemu/tests/win_sigverif.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_test
+from virttest import utils_misc
+from virttest.utils_windows import system
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    sigverif test:
+    1) Boot guest with related virtio devices
+    2) Run sigverif command(an autoit script) in guest
+    3) Open sigverif log and check whether driver is signed
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login()
+
+    driver_name = params["driver_name"]
+    session = utils_test.qemu.windrv_check_running_verifier(session, vm,
+                                                            test, driver_name)
+
+    run_sigverif_cmd = utils_misc.set_winutils_letter(
+        session, params["run_sigverif_cmd"])
+    sigverif_log = params["sigverif_log"]
+    check_sigverif_cmd = params["check_sigverif_cmd"] % driver_name
+    clean_sigverif_cmd = params["clean_sigverif_cmd"]
+
+    error_context.context("Run sigverif in windows guest", logging.info)
```

Comment:
For safety reason, suggest trying remove the output file before running the command.
```
session.cmd(clean_sigverif_cmd, ignore_all_errors=True)
```